### PR TITLE
Single article view

### PR DESCRIPTION
**Pull request by serhii-fsi**
# Task 5 - CORE: View an individual article

-   [x] Add ArticlePage component to the router.
-   [x] Implement ArticlePage component.
-   [x] Implement Article component and loader.
-   [ ] Implement error handling.
-   [ ] Add better styles (later)
    -   [ ] How are you going to make use of the space to display the data?
    -   [ ] How can you display the information in an engaging and accessible way?
    -   [ ] How will this view change with screens of different sizes?

### CODE REVIEW

File: src/components/Article.jsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import { useParams } from "react-router-dom";
+import { fetchArticle } from "../modules/api";
+
+export default function Article() {
+    const { article_id } = useParams();
+    const [article, setArticle] = useState([]);
+    const [isLoading, setIsLoading] = useState(true);
+
+    useEffect(() => {
+        if (article_id >= 0) {
```
Comment (hevdevs) [inline src/components/Article.jsx:11]: Logic like this id check should be handled on the backend in your testing so unnecessary here

File: src/App.css [hunk 2]
```diff
@@ -60,3 +60,37 @@
 .article-tile-li {
     padding: 2px 10px 2px 10px;
 }
+
+/* ArticlePage */
+.article-page-main {
```
Comment (hevdevs) [inline src/App.css:65]: Foundations of responsive styling looking good

**Review by hevdevs — COMMENTED** [review]
Good work so far Serhii 😄

File: src/components/Article.jsx (hunk 1 above)
Comment (serhii-fsi) [reply src/components/Article.jsx:11]: Thank you very much! Really good point - single-responsibility principle, and also I need an error from the server  to display it!

**Pull request merged**